src/fprime/fbuild/builder.py

```diff
@@ -7,21 +7,20 @@
 from pathlib import Path
 from typing import Iterable, List, Union
 
+# Forces targets into existence
+import fprime.fbuild.target_definitions  # lgtm[py/unused-import]
```

Comment:
This import here is quite special and I wonder if we should leave it at the very bottom of the imports, to make it obvious/easily identifiable.

Reply:
Also, we should probably simply revise that pattern.

Reply:
We agreed this was ok, but revision of the pattern is advisable.